app/presenters/return-requirements/check/returns-requirements.presenter.js

```diff
@@ -39,6 +47,32 @@ function _abstractionPeriod (abstractionPeriod) {
   return `From ${startDate} to ${endDate}`
 }
 
+function _agreementsExceptions (agreementsExceptions) {
+  if (agreementsExceptions[0] === agreementsExceptionsText.none) {
+    return 'None'
+  }
+
+  const formattedExceptions = agreementsExceptions.map((exception) => {
+    return agreementsExceptionsText[exception]
+  })
+
+  if (formattedExceptions.length === 1) {
+    return formattedExceptions[0]
+  } else if (formattedExceptions.length === 2) {
+    return formattedExceptions.join(' and ')
+  } else {
+    return _formatExceptionsToSerialCommaList(formattedExceptions)
+  }
```

Comment:
Me and my guard clauses 😁 

```suggestion
  if (formattedExceptions.length === 1) {
    return formattedExceptions[0]
  }
    
  if (formattedExceptions.length === 2) {
    return formattedExceptions.join(' and ')
  }
    
  return _formatExceptionsToSerialCommaList(formattedExceptions)
```